package.xml

```diff
@@ -98,6 +98,9 @@
   <!-- Navigation -->
   <depend>smach_ros</depend>
 
+  <!-- Embedded Software -->
+  <!-- <depend>python3-odrive-pip</depend> -->
```

Comment:
Not sure what to do with this. I feel like it should belong here, but if I uncomment it, it fails the CI check.

Reply:
can you link me that CI failure

Reply:
https://github.com/umrover/mrover-ros/actions/runs/2915947005

Reply:
I'm guessing there are some apt packages that need to get added to the docker container. 

I doubt it'll make a difference but they seem to recommend run-depend, so try that on the off chance that it does something special to resolve python deps: http://docs.ros.org/en/jade/api/catkin/html/howto/format1/python_module_dependencies.html.

Reply:
disregard the suggestion for run-depend, I fixed this by adding `python3-pip` to the container image on Dockerhub